en/admin/db_manage.rst

```diff
@@ -57,9 +57,13 @@ The volumes of CUBRID database are classified as permanent volume, temporary vol
 *   In the permanent volumes,
 
     *   there are data volumes, that usually store permanent data, but can also store temporary data.
-    *   there are log volumes, that can be further classified as: one active log, archive logs and one background archiving log.
+    *   there is the active log volume, that includes late changes of log records.
     
-*   In the temporary volumes, only temporary data is stored.
+*   In the temporary volumes,
+
+    *   there are temporary temp volumes, that store temporary data.
```

Comment:
is it right to use 'that' after comma?

Reply:
@charischau Charis, How do you think about this?

Reply:
In my point of view, 'that' after comma might be the correct expression. 
(reference: https://jakubmarian.com/comma-before-that-and-which/). 

I would recommend remove all comma before 'that' or 'which'. :)
```suggestion
    *   there are temporary temp volumes that store temporary data.
```

Reply:
Thank you! I will apply it :)